pom.xml

```diff
@@ -9,13 +9,13 @@
 	<packaging>jar</packaging>
 	<groupId>org.webjars</groupId>
 	<artifactId>oscar</artifactId>
-	<version>15.0.1-RC1-SNAPSHOT</version>
+	<version>15.0.2-RC1</version>
```

Comment:
Sorry about that.  Just getting used to how this works.  I'd seen the -SNAPSHOT on previous commits but didn't understand why it was there.  Updated now.